packages/patternfly-3/patternfly-react-wooden-tree/src/components/TreeView/TreeViewExample.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import TreeView from './index';
+
+const data = [
+  {
+    text: 'Parent 1',
+    icon: 'fa fa-folder',
+    nodes: [
+      {
+        text: 'Child 1',
+        nodes: [{ text: 'Grandchild 1' }, { text: 'Grandchild 2' }]
+      },
+      {
+        text: 'Child 2',
+        nodes: [{ text: 'Grandchild 1' }, { text: 'Grandchild 2' }]
+      }
+    ]
+  },
+  {
+    text: 'Parent 2',
+    icon: 'fa fa-folder',
+    nodes: [{ text: 'Child 1' }]
+  },
+  {
+    text: 'Parent 3',
+    icon: 'fa fa-folder'
+  },
+  {
+    text: 'Parent 4',
+    icon: 'fa fa-folder'
+  },
+  {
+    text: 'Parent 5',
+    icon: 'fa fa-folder'
+  }
+];
+
+const actionMapper = {
+  'state.expanded': TreeView.nodeExpanded,
```

Comment:
I really like this actionmapper! Much better than to have if or switch. Just a nitpick can we add these as consts?
```JS
const EXPANDED = 'state.expanded';
const CHECKED = 'state.checked';
const DISABLED = 'state.disabled';
const SELECTED = 'state.selected';
const NODES = 'nodes';
const LOADING = 'loading':

const actionMapper = {
 [EXPANDED]: TreeView.nodeExpanded,
 [CHECKED]: TreeView.nodeChecked,
 [DISABLED]: TreeView.nideDisabled,
 [SELECTED]: TreeView.nodeSelected
 [NODES]: TreeView.nodeChildren,
 [LOADING]: TreeView.nodeLoading
}
```